pkg/crypto/hash_test.go

```diff
@@ -0,0 +1,66 @@
+package crypto
+
+import (
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestHash(t *testing.T) {
+	cases := []struct {
+		name           string
+		input          []interface{}
+		expectedOutput string
+		expectedError  error
```

Comment:
is there any way to cause an error in order to test it? This field is unused.

Reply:
I added a case where the json encoder fails.